docs/sensors/lidar_2d/hokuyo_ust30_lx.mdx

```diff
@@ -15,10 +15,9 @@ import Support from "/components/support.mdx";
 
 ## Item Numbers
 
-| Description                      |                         CPR item                         |
-| :------------------------------- | :------------------------------------------------------: |
-| Hokuyo UST20-LX sensor           | [027584](/assets/pdf/clearpath_robotics_027584-TDS1.pdf) |
-| Hokuyo UST20-LX sensor, PACS kit |                                                          |
+| Description            |                         CPR Item                         | Sales Kit | Installation Kit, With Fasteners |
+| :--------------------- | :------------------------------------------------------: | :-------: | :------------------------------: |
+| Hokuyo UST30-LX sensor | [027584](/assets/pdf/clearpath_robotics_027584-TDS1.pdf) |           |                                  |
```

Comment:
Same note as above

Reply:
I have added the kit item numbers for the UST30-LX.